oktapam/data_source_security_policies_test.go

```diff
@@ -107,9 +107,11 @@ resource "oktapam_security_policy" "test_ds_security_policies_1" {
 			}
 			principal_account_rdp {
 				enabled = true
+				admin_level_permissions = true
 			}
 			principal_account_ssh {
 				enabled = true
+				admin_level_permissions = true
```

Comment:
Add at least one scenario where enabled = true and admin_level_permissions=false or admin_level_permissions is not even provided.

Reply:
I don't see any assertion done here for these properties - https://github.com/okta/terraform-provider-oktapam/blob/0f61daa7bae7f1056d06c4aeaab1b125a9867fcb/oktapam/data_source_security_policy_test.go#L35

Reply:
added